aries_staticagent/crypto.py

```diff
@@ -39,11 +40,13 @@ def bytes_to_b64(val: bytes, urlsafe=False) -> str:
     return base64.b64encode(val).decode("ascii")
 
 
+@lru_cache(maxsize=16)
```

Comment:
do we anticipate frequent conversion of the same values?

Reply:
While unpacking a message, multiple keys are converted from one format to another; after returning the unpacked message, there's a good chance the same keys might be used for verifying a signature or other crypto operations which would again necessitate converting back and forth. So a small cache seemed like a fair idea, especially given how simple it is to implement. I'm not married to the idea though.

Reply:
Added a comment